go/vt/vttablet/tabletmanager/rpc_replication.go

```diff
@@ -752,6 +770,10 @@ func (tm *TabletManager) PromoteReplica(ctx context.Context) (string, error) {
 		return "", err
 	}
 
+	// Clear replication sentinel flag for this master,
+	// or we might block replication the next time we demote it
+	tm.replManager.setReplicationStopped(false)
```

Comment:
Why is a `MASTER` tablet marked with `false`? I'd again think we need `true` here?

Reply:
Same logic as above;  we want to remove the on-disk flag that will prevent vttablet from "fixing" replication at some later point in time, if this master ever becomes a replica again.